drivers/hmis/app/graphql/types/hmis_schema/query_type.rb

```diff
@@ -157,8 +157,9 @@ def pick_list(pick_list_type:, relation_id: nil)
     field :current_user, Application::User, null: true
 
     access_field do
-      can :edit_clients, root: true
-      can :view_clients, root: true
+      Hmis::Role.permissions_with_descriptions.keys.each do |perm|
+        can perm, field_name: perm, method_name: perm, root: true
+      end
```

Comment:
This differs from what we discussed and outlined [in the google doc](https://docs.google.com/document/d/1wU7i8DUl2-y6PpPcJaWq46dHUvNvwmiMspTwSh-vdzA/edit#heading=h.2n3ogufrqrrv). Sorry, I must have forgotten to link to this section from the PT story.

But now looking at the code, I think this is better anyway. It doesn't seem worth it to maintain a custom global access gql type with custom fields (like "canViewClientDashboard") that just resolve to these permissions.